notification/index.azureFunction.js

```diff
@@ -0,0 +1,66 @@
+import notificationHandler from './src/handler/notification/notification.handler.js'
+import { getLogger } from './src/utils/logger.js'
+import utils from './src/utils/commons.js'
+import { getErrorCause, isRecoverableError } from './src/utils/error-utils.js'
+import { getCtpProjectConfig, getAdyenConfig } from './src/utils/parser.js'
+
+const logger = getLogger()
+
+export const azureExtensionTrigger = async function (context, event) {
+  const { notificationItems } = event.body
```

Comment:
what if `body` object is absent in `event`?

Reply:
Then we don't have notificationItems, So it will return `No notification received`.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/993/commits/e980819c0531bad932f8ab1bf65462a86dddf7fc

I added an additional check.